integration/test-integration/src/test/resources/openAPIs/trailing_slash_openAPI.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Trailing Slash
+servers:
+  - url: http://mockBackend:2383/v2
+x-wso2-basePath: /trailing-slash
+paths:
+  /echo-full/findByStatus:
```

Comment:
Shall we check with another resource with a trailing slash and see the behavior?
ex: `/foo/ `

Reply:
I updated policy and default API testcases as you suggested. They call other paths that are not `/echo-full`. Changes made to Default API testcase covers all three cases. https://github.com/wso2/product-microgateway/pull/2878/commits/fc9d1952fb8ad3d82c7fd091e0ff550193152fa6 Will these do?

Reply:
The idea here is to like verify if the swagger resource has a trailing `/` and when we are forwarding it to the backend, whether we append that remaining slash.
As I saw, all the tests are like there is a resource with no slash and we are checking the both options. But nothing covers the pattern I tell right?

Reply:
Thanks for the explanation. Got the idea. Will update.

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2878/commits/dc1a2a713cc236684b6626c495fd64319604069a